website/docs/components/flyout/partials/guidelines/guidelines.md

```diff
@@ -81,28 +81,12 @@ Don't use a form in a Flyout, whether the function is creating an object, or upd
 
 ## Size
 
-Medium
+The available sizes are medium (default) and large.
 
-<Hds::Flyout id="size-medium-flyout" class="doc-flyout-demo" open as |F|>
-  <F.Header>Medium Flyout</F.Header>
-  <F.Body>
-    <p class="hds-typography-body-300 hds-foreground-primary">Flyout content</p>
-  </F.Body>
-</Hds::Flyout>
-
-Large
-
-<Hds::Flyout @size="large" id="size-large-flyout" class="doc-flyout-demo" open as |F|>
-  <F.Header>Large Flyout</F.Header>
-  <F.Body>
-    <p class="hds-typography-body-300 hds-foreground-primary">Flyout content</p>
-  </F.Body>
-</Hds::Flyout>
+![A medium (480px wide) and large (720px wide) Flyout side by side.](/assets/components/flyout/flyout-sizes.png)
```

Comment:
![Screenshot 2024-09-05 at 10 49 25 AM](https://github.com/user-attachments/assets/a466c923-523b-4a3d-92ff-d96bee86adb1)

Doesn't look like an issue in the rendered page. I think the sizes are kinda necessary because it lets people know what the difference is between the sizes (sighted users can get the gist of how different the sizes are by looking).